Projekty/Projekt1/Fijalkowski_Krzyzinski_Nocon/EDA.ipynb

```diff
@@ -0,0 +1,2081 @@
+{
```

Comment:
<p>Jest to słuszna uwaga. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>